### PR TITLE
Sign up & Sign in

### PR DESCRIPTION
**Pull request by Said-laasri**
1. Add axios library for HTTP request: `npm install Axios`

2. Create Send Post request using Axios with Email and Password to Login the user and on success save **JWT token** in Local Storage then redirect to `/chat` route to chat page.

3. Create send Post request using Axios with Username, Email, and Password to Register a new user and on success save **JWT token** in Local Storage then redirect to the `/chat` route to the chat page.

### CODE REVIEW

File: src/entryPoint/Login.tsx [hunk 1]
```diff
@@ -45,16 +47,48 @@ const LogIn = () => {
...
+const navigate = useNavigate()
+  const handleSubmit = async (e) => {
     e.preventDefault();
 
     validateForm();
 
     if (isFormValid) {
-      // Perform further actions, such as making API requests or handling form submission
+      try {
+        const response = await axios.post('http://localhost:8000/api/v1/users/log-in', {
```
Comment (frankstepanski) [inline src/entryPoint/Login.tsx:58]: We will eventually change the hard-coded domain, but totally fine for now.

**Review by frankstepanski — APPROVED** [review]
Looks good. Great job.

File: src/entryPoint/Login.tsx (hunk 1 above)
Comment (Said-laasri) [reply src/entryPoint/Login.tsx:58]: yeah with optimization will change all of that hard-coded stuff
thanks for the review  @frankstepanski

**Comment by Said-laasri** [comment]
> Looks good. Great job.

thanks

**Pull request merged**